src/library/scala/collection/immutable/Vector.scala

```diff
@@ -81,8 +81,8 @@ object Vector extends StrictOptimizedSeqFactory[Vector] {
   }
 
   private val defaultApplyPreferredMaxLength: Int =
-    try System.getProperty("scala.collection.immutable.Vector.defaultApplyPreferredMaxLength",
-      "250").toInt
+    try new StringOps(System.getProperty("scala.collection.immutable.Vector.defaultApplyPreferredMaxLength",
```

Comment:
```suggestion
    // explicit StringOps to avoid initialization cycle with Predef (scala/bug#13009)
    try new StringOps(System.getProperty("scala.collection.immutable.Vector.defaultApplyPreferredMaxLength",
```